.history/person_20230130194757.rb

```diff
@@ -0,0 +1,14 @@
+class Person
+    def initialize(type, number_of_legs, name = 'Unknown')
+      @id = Random.rand(1..1000)
+      @name = name
+      @age = age
+    end
+  
+    def speak
+      "grrrr"
+    end
+  
+    attr_accessor :name, :age
+    attr_reader :id
```

Comment:
- [OPTIONAL] It would be nice to delete the _**.history**_ directory. This way, you would be implementing the **_You Aint Gonna Need It_** principle in programming.  Kindly delete it. 💻 😎